lib/resources/collection/index.js

```diff
@@ -448,7 +448,7 @@ Collection.prototype.save = function (ctx, fn) {
 
   domain.changed =  function (property) {
```

Comment:
I still think it's somewhat unintuitive and can lead to issues.

How about changedAny('fieldA', 'fieldB') and changedAll('fieldA', 'fieldB') as two separate methods? It would be just as simple to implement.

I can think about things like custom logic for notifying an admin if ALL the specified fields are changed, but I can definitely see the ANY case being just as useful.

As an example, I could have:
```
if (changedAll('availabilityStatus', 'availabilityUntil')) {
  ..logic
} else {
  cancel('both availabilityStatus and availabilityUntil to be updated simultaneously');
}
```